ecosystem/typescript/sdk/src/coin_client.ts

```diff
@@ -52,28 +52,19 @@ export class CoinClient {
     extraArgs?: OptionalTransactionArgs & {
       // The coin type to use, defaults to 0x1::aptos_coin::AptosCoin
       coinType?: string;
-      // If set, create the `receiver` account if it doesn't exist on-chain.
-      // This is done by calling `0x1::aptos_account::transfer` instead, which
-      // will create the account on-chain first if it doesn't exist before
-      // transferring the coins to it.
-      createReceiverIfMissing?: boolean;
     },
   ): Promise<string> {
     // If none is explicitly given, use 0x1::aptos_coin::AptosCoin as the coin type.
     const coinTypeToTransfer = extraArgs?.coinType ?? APTOS_COIN;
 
-    // If we should create the receiver account if it doesn't exist on-chain,
-    // use the `0x1::aptos_account::transfer` function.
-    const func = extraArgs?.createReceiverIfMissing ? "0x1::aptos_account::transfer" : "0x1::coin::transfer";
```

Comment:
that looks like a type of a breaking change in the underline implementation, where users don't `createReceiverIfMissing` in their app - now have to create a user..

Reply:
aptos_account::transfer_coins implicitly creates the account if it doesn't exist

Reply:
correct, but the existing functionality giving the user to choose whether they want to create the account if it doesnt exist (hence the use of `0x1::coin::transfer`) - I am just saying that this decision that is now left for the user is gone, and introduce an underlying implementation change

Reply:
I added createReceiverIfMissing back.